techniques/system/common/1.0/update.st

```diff
@@ -428,3 +451,10 @@ body perms u_mog(mode,user,group)
   owners => { "${user}" };
   groups => { "${group}" };
 }
+
+
+bundle edit_line rudder_update_lines_present(lines)
```

Comment:
all other bodys/bundle have a more specific name. I fear collision with a name that common
Could you name it u_rudder_update_lines_present ?

Reply:
Where is this used